integration-tests/src/test/java/com/datastax/oss/driver/api/querybuilder/JsonInsertIT.java

```diff
@@ -0,0 +1,253 @@
+import com.datastax.oss.driver.api.testinfra.session.SessionUtils;
+import com.datastax.oss.driver.categories.ParallelizableTests;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Duration;
+import java.util.List;
+import java.util.Objects;
+import org.assertj.core.api.Assertions;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+
+@Category(ParallelizableTests.class)
+public class JsonInsertIT {
+  private static final CcmRule ccmRule = CcmRule.getInstance();
+  private static final JacksonJsonCodec<User> JACKSON_JSON_CODEC =
+      new JacksonJsonCodec<>(User.class);
+
+  private static SessionRule<CqlSession> sessionRule =
+      SessionRule.builder(ccmRule)
+          .withConfigLoader(
+              SessionUtils.configLoaderBuilder()
+                  .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(30))
+                  .build())
+          .build();
+
+  @ClassRule public static TestRule chain = RuleChain.outerRule(ccmRule).around(sessionRule);
+
+  @BeforeClass
+  public static void setup() {
+    sessionRule
+        .session()
+        .execute("CREATE TABLE json_jackson_row(id int PRIMARY KEY, name text, age int)");
+  }
+
+  @After
+  public void clearTable() {
+    sessionRule.session().execute("TRUNCATE TABLE json_jackson_row");
+  }
+
+  @Test
+  public void should_insert_string_as_json_using_simple_statement() {
+    // given a simple statement
+    try (CqlSession session = sessionWithCustomCodec()) {
+      String jsonUser = "{ \"id\": 2, \"name\": \"Alice\", \"age\": 3 }";
+      Statement stmt = insertInto("json_jackson_row").json(jsonUser).build();
+
+      // when
+      session.execute(stmt);
+
+      // then
+      String jsonUserResult =
+          session
+              .execute(selectFrom("json_jackson_row").json().all().build())
+              .all()
+              .get(0)
+              .getString(0);
+
+      assertThat(jsonUserResult).contains("\"id\": 2");
+      assertThat(jsonUserResult).contains(" \"name\": \"Alice\"");
+      assertThat(jsonUserResult).contains("\"age\": 3");
+    }
+  }
+
+  @Test
+  public void should_insert_json_using_prepare_statement() {
+    // given prepare statement
+    try (CqlSession session = sessionWithCustomCodec()) {
+      User user = new User(2, "bob", 35);
+      PreparedStatement pst =
+          session.prepare(insertInto("json_jackson_row").json(bindMarker("user")).build());
+
+      // when
+      session.execute(pst.bind().set("user", user, User.class));
+
+      // then
+      List<Row> rows = session.execute(selectFrom("json_jackson_row").json().all().build()).all();
+      Assertions.assertThat(rows.get(0).get(0, User.class)).isEqualTo(user);
```

Comment:
Nit: remove `Assertions.`, the method is statically imported.
(this happens in a few other places)